hazelcast/src/test/java/com/hazelcast/jet/impl/connector/WriteJdbcPTest.java

```diff
@@ -176,22 +173,25 @@ public void test() throws SQLException {
     public void test_supplied_closeable_datasource_is_closed() throws SQLException {
         Pipeline p = Pipeline.create();
 
+        CopyOnWriteArrayList<HikariDataSource> hikariDataSourceList = new CopyOnWriteArrayList<>();
```

Comment:
Why we couldn’t simply use local the local `hikariDataSource` variable here?

Reply:
There are two members in the cluster. Each one is calling the supplier to create a datasource and these are added to the list. At the end of the test there will be two entries in the list